README.md

```diff
@@ -1,11 +1,283 @@
+
+## How the SDK works <a name="how"></a>
+At a very basic level, the OMSDK controls only the video frame. Because of this, you are completely in control of your app’s design and UX (look and feel). You can control whether videos play in a small view, in-place over a thumbnail image, or at full-screen. Your app also has complete control over device rotation, use of view/navigation controllers, scrollers, and any transitions between them. The SDK does not dictate any overall visual design or behavior on your app.
+
+If you choose to use the SDK’s built-in default player controls UX implementation, then that part of the video UX is imposed on you. All controls rendering is also done within the frame you provide for the video. Regardless of which controls UX you use, we currently do not allow any customization or overriding of the ads playback UX (which is different from the normal video playback UX), so that visual interface is dictated, and you cannot override it. Future customization options are planned here.
+
+To play a video, you follow these very basic steps:
+1. Initialize an instance of the OMSDK
+2. Initialize a new `Player` object with a video ID
+3. Attach it to a view controller
+4. Play it
+
+That’s it!
+
+Behind the scenes, what happens is this … the initialization of an instance of the SDK takes your app’s bundle ID, and passes it to our back-end micro services to authenticate your application for video playback within the O2 video platform. The server passes back all the necessary playback and authentication information down to that instance of the SDK, for use during it’s lifespan. When you construct a new `Player` object from that SDK instance, it communicates with our micro services to obtain all the necessary video metadata (e.g., thumbnails and video URLs, duration, etc.). This Player`` object will play and replay the associated video until deinitialized.
+
+More specifically, before a video plays, the SDK’s Ads Engine tries to fulfill a pre-roll ad. While the request for an ad is being processed, the video starts buffering in the background. If an ad is returned in a reasonable amount of time, the `Player` plays the ad using the built-in ads UX. When the ad finishes, the video playback begins. If no ad is to be shown, or the ad request times out, the video playback begins directly.
+
+The runtime circumstances and algorithm for getting an ad or not, are not in the scope of this documentation. Suffice to say, there are many considerations to this (e.g., content owner/seller rules, geolocation, frequency capping, etc.). For more information and details on how ads are served to the OMSDK, please email the [Video Support Team](mailto:video.support@oath.com).
+
+**Note**: The SDK only operates with an active network connection – without it, you will not be able to do anything.
+
+## Default (Player) Controls UX <a name="ux"></a>
+The default player controls UX contains the following elements:
```

Comment:
Insert pic here - better than 10 lines of text.

Reply:
Good idea ... if I could find a good pic, that's accurate, I'd annotate it.  Still think the text would be relevant.  Be hard to show everything especially since live, 360, and being in array or playlist is hard to code at same time...